uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-calendar-legend.html

```diff
@@ -39,7 +39,7 @@ <h2 class="panel-title chr_leave-calendar__legend__title" role="button" ng-click
             <span class="chr_leave-calendar__legend__label">{{absenceType.title}} specific time</span>
           </div>
         </div>
-        <div class="col-xs-6 col-md-3">
+        <div class="col-xs-6 col-md-3" ng-if="absenceType.calculation_unit_name === 'days'">
```

Comment:
`ng-if` will hide remove the element from the DOM. Are you sure you want to do so?